{"ygPerms":{"resourceCapabilityList":[{"resourceType":"GROUP","capabilities":[{"name":"READ"},{"name":"JOIN"}]},{"resourceType":"PHOTO","capabilities":[]},{"resourceType":"FILE","capabilities":[]},{"resourceType":"MEMBER","capabilities":[]},{"resourceType":"LINK","capabilities":[]},{"resourceType":"CALENDAR","capabilities":[]},{"resourceType":"DATABASE","capabilities":[]},{"resourceType":"POLL","capabilities":[]},{"resourceType":"MESSAGE","capabilities":[{"name":"READ"}]},{"resourceType":"PENDING_MESSAGE","capabilities":[]},{"resourceType":"ATTACHMENTS","capabilities":[{"name":"READ"}]},{"resourceType":"PHOTOMATIC_ALBUMS","capabilities":[]},{"resourceType":"MEMBERSHIP_TYPE","capabilities":[]},{"resourceType":"POST","capabilities":[{"name":"READ"}]},{"resourceType":"PIN","capabilities":[]}],"groupUrl":"groups.yahoo.com","intlCode":"us"},"comscore":"pageview_candidate","ygData":{"userId":27164717,"authorName":"didomax77","from":"&quot;didomax77&quot; &lt;prmaster@...&gt;","profile":"didomax77","replyTo":"LIST","senderId":"vTexW7yLTiS73RWOgObwcbLf3A6miGjyl5AVbKsnxuSH2p8iY--gRQ0Nki8dXs8WGjYd9pfyYY58p9RfMQ5kM6AdO7PJcX-phLmD","spamInfo":{"isSpam":false,"reason":"12"},"subject":"segmentation fault in flame2.py","postDate":"1126627133","msgId":983,"canDelete":false,"contentTrasformed":false,"systemMessage":true,"headers":{"messageIdInHeader":"PGRnNnN2dCszdTAxQGVHcm91cHMuY29tPg=="},"prevInTopic":0,"nextInTopic":0,"prevInTime":982,"nextInTime":984,"topicId":983,"numMessagesInTopic":1,"msgSnippet":"Good morning Prof. Goodwin I am testing the premixed flame example with python interface (flame2.py) but I get a segmentation fault error at the following ","rawEmail":"Return-Path: &lt;prmaster@...&gt;\r\nReceived: (qmail 67858 invoked from network); 14 Sep 2005 01:05:10 -0000\r\nReceived: from unknown (66.218.66.217)\n  by m32.grp.scd.yahoo.com with QMQP; 14 Sep 2005 01:05:10 -0000\r\nReceived: from unknown (HELO n10a.bulk.scd.yahoo.com) (66.94.237.44)\n  by mta2.grp.scd.yahoo.com with SMTP; 14 Sep 2005 01:05:10 -0000\r\nComment: DomainKeys? See http://antispam.yahoo.com/domainkeys\r\nReceived: from [66.218.69.4] by n10.bulk.scd.yahoo.com with NNFMP; 14 Sep 2005 01:05:05 -0000\r\nReceived: from [66.218.66.85] by mailer4.bulk.scd.yahoo.com with NNFMP; 14 Sep 2005 01:05:05 -0000\r\nX-Sender: prmaster@...\r\nX-Apparently-To: cantera@yahoogroups.com\r\nReceived: (qmail 26776 invoked from network); 13 Sep 2005 15:59:05 -0000\r\nReceived: from unknown (66.218.66.167)\n  by m35.grp.scd.yahoo.com with QMQP; 13 Sep 2005 15:59:05 -0000\r\nReceived: from unknown (HELO n20a.bulk.scd.yahoo.com) (66.94.237.49)\n  by mta6.grp.scd.yahoo.com with SMTP; 13 Sep 2005 15:59:04 -0000\r\nComment: DomainKeys? See http://antispam.yahoo.com/domainkeys\r\nReceived: from [66.218.66.58] by n20.bulk.scd.yahoo.com with NNFMP; 13 Sep 2005 15:58:53 -0000\r\nReceived: from [66.218.66.79] by mailer7.bulk.scd.yahoo.com with NNFMP; 13 Sep 2005 15:58:53 -0000\r\nDate: Tue, 13 Sep 2005 15:58:53 -0000\r\nTo: cantera@yahoogroups.com\r\nMessage-ID: &lt;dg6svt+3u01@...&gt;\r\nUser-Agent: eGroups-EW/0.82\r\nMIME-Version: 1.0\r\nContent-Type: text/plain; charset=ISO-8859-1\r\nContent-Length: 420\r\nX-Mailer: Yahoo Groups Message Poster\r\nX-Yahoo-Newman-Property: groups-system\r\nX-eGroups-Msg-Info: 1:12:0:0\r\nFrom: &quot;didomax77&quot; &lt;prmaster@...&gt;\r\nSubject: segmentation fault in flame2.py\r\nX-Yahoo-Group-Post: member; u=27164717; y=e1vZVg_PIsINNQlyQ-87M08CHYkuVyrq31neT2jAWA0M6GP7\r\nX-Yahoo-Profile: didomax77\r\nX-eGroups-Approved-By: dggoodwin &lt;dgoodwin@...&gt; via web; 14 Sep 2005 01:05:02 -0000\r\n\r\nGood morning Prof. Goodwin\n  I am testing the premixed flame example with python interface\n(flame2.py) but I get a &quot;segmentation fault&quot; error at the following\ninstruction:\n\nf.setTimeStep(1.0e-5, [1, 2, 5, 10, 20])\n\nIf I comment out this line I get the solution w/o any problem. I have\nchecked the python documentation, the syntax seems to be correct.\nI am running on a Linux machine, FC4, python 2.4.1.\nAny idea?\nThanks\n\n\n\n\n\n"}}